test/browser_tests/conf.js

```diff
@@ -1,32 +1,118 @@
 'use strict';
 
-var env = require( './environment.js' );
-
-exports.config = {
-  multiCapabilities: [
-    {
-      browserName: 'chrome',
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    {
-      browserName: 'firefox',
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    // Large Screen only tests
-    {
-      browserName:   'chrome',
-      specs:         [ 'spec_suites/large_screen/*.js' ],
-      chromeOptions: {
-        args: [ '--lang=en',
-                '--window-size=1200,900' ]
+var settings = require( './settings.js' );
+var defaultSuites = require( './default-suites.js' );
+
+function _paramIsSet( param ) {
+  return typeof param !== 'undefined';
+}
+
+/*
+* Params that need to be passed to protractor config.
+*/
+function _retrieveProtractorParams( params ) { // eslint-disable-line complexity, no-inline-comments, max-len
+  var parsedParams = {};
+
+  var checkParam = params.specs;
+  if ( _paramIsSet( checkParam ) ) {
+    parsedParams.specs = settings.specsBasePath + checkParam;
+  }
+
+  checkParam = params.browserName;
+  if ( _paramIsSet( checkParam ) ) { parsedParams.browserName = checkParam; }
+
+  checkParam = params.version;
+  if ( _paramIsSet( checkParam ) ) { parsedParams.version = checkParam; }
+
+  checkParam = params.platform;
+  if ( _paramIsSet( checkParam ) ) { parsedParams.platform = checkParam; }
+
+  return parsedParams;
+}
+
+function _copyParameters( params, capabilities ) { // eslint-disable-line complexity, no-inline-comments, max-len
+  var newCapabilities = [];
+  var injectParams = _retrieveProtractorParams( params );
+  var capability;
+  for ( var i = 0, len = capabilities.length; i < len; i++ ) {
+    capability = capabilities[i];
+    for ( var p in injectParams ) {
+      if ( injectParams.hasOwnProperty( p ) ) {
+        capability[p] = injectParams[p];
       }
     }
-  ],
+    capability.name = settings.testName +
+                      ' ' + capability.specs +
+                      ', running ' +
+                      capability.browserName +
```

Comment:
Ah ok, I see what's happening now. Sorry, there's a lot going on, much easier to follow in an IDE than this one big page.